src/mdbExtensionController.ts

```diff
@@ -72,6 +72,7 @@ export default class MDBExtensionController implements vscode.Disposable {
   _editDocumentCodeLensProvider: EditDocumentCodeLensProvider;
   _exportToLanguageCodeLensProvider: ExportToLanguageCodeLensProvider;
   _participantController: ParticipantController;
+  _surveyShown = false;
```

Comment:
nit: Could we rename this to `_startupNotificationShown`, and then update the logic to be that both of the notification methods check for this, not just the copilot one? That way it's not bound to where we call it in the code. Things may shift around and we might add more notifications in the future so it'll make it a bit more resilient.